.vscode/settings.json

```diff
@@ -1,3 +1,18 @@
 {
-    "python.analysis.typeCheckingMode": "basic"
-}
\ No newline at end of file
+    "python.analysis.typeCheckingMode": "basic",
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.autopep8",
```

Comment:
This and Prettier need to be provided by extensions, which people might not have installed. Click the Settings gear on the extension and select "Add to workspace recommendations" so VSCode will prompt to install them if not already